### ISSUE TITLE
Was not able to build for node v12.4.0

### ISSUE THREAD

**Issue by cryptofuture** [cossacklabs/themis#478]
```
npm install jsthemis

> jsthemis@0.11.2 install /node_modules/jsthemis
> node-gyp rebuild

make: Entering directory '/node_modules/jsthemis/build'
  CXX(target) Release/obj.target/jsthemis/addon.o
In file included from ../addon.cpp:19:0:
../errors.hpp:30:15: error: variable or field ‘Init’ declared void
 void Init(v8::Handle<v8::Object> exports);
               ^~~~~~
../errors.hpp:30:15: error: ‘Handle’ is not a member of ‘v8’
../errors.hpp:30:32: error: expected primary-expression before ‘>’ token
 void Init(v8::Handle<v8::Object> exports);
                                ^
../errors.hpp:30:34: error: ‘exports’ was not declared in this scope
 void Init(v8::Handle<v8::Object> exports);
                                  ^~~~~~~
In file included from ../addon.cpp:20:0:
../secure_cell_context_imprint.hpp:30:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_cell_context_imprint.hpp:30:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:21:0:
../secure_cell_seal.hpp:30:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_cell_seal.hpp:30:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:22:0:
../secure_cell_token_protect.hpp:30:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_cell_token_protect.hpp:30:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:23:0:
../secure_comparator.hpp:32:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_comparator.hpp:32:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:24:0:
../secure_keygen.hpp:32:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_keygen.hpp:32:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:25:0:
../secure_message.hpp:30:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_message.hpp:30:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
In file included from ../addon.cpp:26:0:
../secure_session.hpp:32:26: error: ‘v8::Handle’ has not been declared
     static void Init(v8::Handle<v8::Object> exports);
                          ^~~~~~
../secure_session.hpp:32:32: error: expected ‘,’ or ‘...’ before ‘<’ token
     static void Init(v8::Handle<v8::Object> exports);
                                ^
../addon.cpp:28:18: error: variable or field ‘InitAll’ declared void
 void InitAll(v8::Handle<v8::Object> exports)
                  ^~~~~~
../addon.cpp:28:18: error: ‘Handle’ is not a member of ‘v8’
../addon.cpp:28:35: error: expected primary-expression before ‘>’ token
 void InitAll(v8::Handle<v8::Object> exports)
                                   ^
../addon.cpp:28:37: error: ‘exports’ was not declared in this scope
 void InitAll(v8::Handle<v8::Object> exports)
                                     ^~~~~~~
In file included from ../addon.cpp:17:0:
../addon.cpp:40:23: error: ‘InitAll’ was not declared in this scope
 NODE_MODULE(jsthemis, InitAll)
                       ^
/.node-gyp/12.4.0/include/node/node.h:556:36: note: in definition of macro ‘NODE_MODULE_X’
       (node::addon_register_func) (regfunc),                          \
                                    ^~~~~~~
../addon.cpp:40:1: note: in expansion of macro ‘NODE_MODULE’
 NODE_MODULE(jsthemis, InitAll)
 ^~~~~~~~~~~
jsthemis.target.mk:115: recipe for target 'Release/obj.target/jsthemis/addon.o' failed
make: *** [Release/obj.target/jsthemis/addon.o] Error 1
make: Leaving directory '/node_modules/jsthemis/build'
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/.npm-packages/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:200:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Linux 4.15.0-51-generic
gyp ERR! command "/usr/bin/node" "/.npm-packages/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
```

**Comment by vixentael** [comment]
oh those changes in Node.js again! 😂

thank you, @cryptofuture, we will take a look. 
Could you please specify versions of OS, npm and node that you are using, to make it easier for us to setup test environment.

**Comment by cryptofuture** [comment]
Ubunu

> oh those changes in Node.js again! joy
> 
> thank you, @cryptofuture, we will take a look.
> Could you please specify versions of OS, npm and node that you are using, to make it easier for us to setup test environment.
Ubuntu bionic (18.04.x)
Node.js v12.x from here 
https://github.com/nodesource/distributions#deb
Exact version would be: 
```
nodejs -v
v12.4.0
npm -v
6.9.0
```

**Comment by ilammy** [comment]
This is fixed by #499. Node.js 12 support will be available in the next release.
